products/cisco-ios-xe.md

```diff
@@ -0,0 +1,143 @@
+
+  - releaseCycle: "17.13"
+    releaseDate: 2023-11-30
+    eol: 2027-07-31
+    eoas: 2024-12-31
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/ios-xe-17-13-x-eol.html
+
+  - releaseCycle: "17.12"
+    releaseDate: 2023-07-28
+    eol: 2029-03-31
+    eoas: 2027-09-30
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/ios-xe-17-12-x-eol.html
+
+  - releaseCycle: "17.11"
+    releaseDate: 2023-03-28
+    eol: 2026-11-30
+    eoas: 2024-05-14
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/ios-xe-17-11-x-eol.html
+
+  - releaseCycle: "17.10"
+    releaseDate: 2022-11-30
+    eol: 2026-07-31
+    eoas: 2024-01-03
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/ios-xe-17-10-x-eol.html
+
+  - releaseCycle: "17.9"
+    releaseDate: 2022-07-29
+    eol: 2027-03-31
+    eoas: 2026-09-30
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/ios-xe-17-9-x-eol.html
+
+  - releaseCycle: "17.8"
+    releaseDate: 2022-04-11
+    eol: 2027-11-30
+    eoas: 2023-05-30
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/ios-xe-17-8-x-eol.html
+
+  - releaseCycle: "17.7"
+    releaseDate: 2021-11-30
+    eol: 2027-07-31
+    eoas: 2023-01-30
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/ios-xe-17-7-x-eol.html
+
+  - releaseCycle: "17.6"
+    releaseDate: 2021-07-30
+    eol: 2026-03-31
+    eoas: 2024-09-30
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/ios-xe-17-6-x-eol.html
+
+  - releaseCycle: "17.5"
+    releaseDate: 2021-03-31
+    eol: 2026-11-30
+    eoas: 2022-05-30
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/ios-xe-17-5-x-eol.html
+
+  - releaseCycle: "17.4"
+    releaseDate: 2020-11-30
+    eoas: 2022-01-28
+    eol: 2026-07-31
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/eos-eol-notice-c51-2442351.html
+
+  - releaseCycle: "17.3"
+    releaseDate: 2020-07-31
+    eol: 2027-03-31
+    eoas: 2023-09-30
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/ios-xe-17-3-x-eol.html
+
+  - releaseCycle: "17.2"
+    releaseDate: 2020-03-30
+    eol: 2026-01-31
+    eoas: 2021-07-15
+    link: https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-17/eos-eol-notice-c51-744388.html
+
+  - releaseCycle: "17.1"
+    releaseDate: 2019-11-21
+    eol: 2025-06-30
+    eoas: 2021-03-31
+    link: https://www.cisco.com/c/en/us/support/ios-nx-os-software/ios-xe-amsterdam-17-1-1/model.html
+
+---
+
+> Cisco IOS XE is a network operating system that is based on the Linux kernel.
+> It is used on a variety of Cisco products, including routers, switches, and wireless controllers.
+
```

Comment:
It would be nice to start by mentioning the release cadence, as explained on https://www.cisco.com/c/en/us/products/collateral/ios-nx-os-software/ios-xe-16/bulletin-c25-2378701.html:

> The schedule specifies 3 individual software releases per year at 4 month intervals.